dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -63,8 +64,8 @@ static StaticLinkageChecker create(
   }
 
   /**
-   * Flag on the reachability. This flag controls whether the report excludes the linkage errors
-   * on classes that are not reachable from the entry points of the class usage graph.
+   * Flag that controls whether the report excludes linkage errors on classes that are not reachable
```

Comment:
Can we be explicit about what `true` means? "controls whether it excludes errors that are not reachable" makes my brain hurt.